bazel/repository_locations.bzl

```diff
@@ -50,13 +50,10 @@ REPOSITORY_LOCATIONS = dict(
         commit = "43863938377a9ea1399c0596269e0890b5c5515a",
         remote = "https://github.com/google/googletest",
     ),
-    # TODO(htuch): Protobuf can switch back to a point release http_archive at the next
-    # release (> 3.4.1), we need HEAD proto_library support and
-    # https://github.com/google/protobuf/pull/3761.
     com_google_protobuf = dict(
-        sha256 = "5d4551193416861cb81c3bc0a428f22a6878148c57c31fb6f8f2aa4cf27ff635",
-        strip_prefix = "protobuf-c4f59dcc5c13debc572154c8f636b8a9361aacde",
-        urls = ["https://github.com/google/protobuf/archive/c4f59dcc5c13debc572154c8f636b8a9361aacde.tar.gz"],
+        sha256 = "0cc6607e2daa675101e9b7398a436f09167dffb8ca0489b0307ff7260498c13c",
```

Comment:
Can you also switch at https://github.com/envoyproxy/data-plane-api/blob/master/WORKSPACE#L10?

Reply:
Done: https://github.com/envoyproxy/data-plane-api/pull/244